packages/common/lib/common/tools.js

```diff
@@ -1278,7 +1278,7 @@ async function getRepositoryFileAsync(url, hash, force, _actualRepo) {
             data = _actualRepo;
         } else {
             const agent = `${module.exports.appName}, RND: ${randomID}, Node:${process.version}, V:${
-                require('../../package.json').version
+                require('../../../package.json').version
```

Comment:
We could/should `require("@iobroker/common/package.json")` (or however this module is called) to avoid juggling with varying levels of `../` all the time.